implementations/rust/ockam/ockam/src/abac/mod.rs

```diff
@@ -0,0 +1,21 @@
+//! Attribute Based Access Control
+
+pub mod access_control;
```

Comment:
I'm leaning towards a separate crate as well. Unless anyone thinks different by the end of the week I think I'm going to do that.

Reply:
Moved to `ockam_examples/example_projects/access_control`.

Reply:
I also took a look at moving `IdentityIdentifier` to `ockam_core` but things got hairy very quickly.

On the balance of things I think we're better off giving it it's own crate.

I've moved the implementation to `ockam_abac`. It's pretty minimal after moving the bits for the `AccessControl` out but it will probably start expanding rapidly once we start implementing the service for Elixir and integrating 3rd party sources etc.